examples/tour/src/main.rs

```diff
@@ -730,16 +729,16 @@ impl Language {
     }
 }
 
-impl From<Language> for &str {
-    fn from(language: Language) -> &'static str {
-        match language {
+impl From<Language> for String {
+    fn from(language: Language) -> String {
+        String::from(match language {
```

Comment:
If I'm not mistaken, such variant could be implemented with just `#[derive(Display)]` and `language.to_string()`.

Reply:
Yes, that'd be cleaner. I'll change the example. Thanks!